packages/block-editor/src/components/inserter/style.scss

```diff
@@ -32,10 +32,11 @@ $block-inserter-search-height: 38px;
 	height: 100%;
 	display: flex;
 	width: auto;
+	min-width: 400px;
```

Comment:
What about viewports which are narrower than 400px?

![image](https://user-images.githubusercontent.com/1779930/74493176-50c3d780-4e9f-11ea-83cd-ebe60dfefb2f.png)

Reply:
Whoops, that needs to be in a media query.